front/src/routes/settings/settings-background-jobs/index.js

```diff
@@ -13,6 +13,9 @@ class SettingsSystem extends Component {
       const skip = page * NUMBER_OF_JOBS_PER_PAGE;
       const jobs = await this.props.httpClient.get(`/api/v1/job?take=${NUMBER_OF_JOBS_PER_PAGE}&skip=${skip}`);
 
+      // We replace "2023-01-23 14:20:23.594" +00:00 by "2023-01-23T14:20:23.594+00:00" to be Firefox compatible
+      jobs.forEach(job => (job.created_at = job.created_at.replace(' ', 'T').replace(' ', '')));
```

Comment:
Good for me, but it's not enough! In this file, there are a few other functions where jobs are added/updated

Jobs are added/updated live in websockets

Reply:
Updated ✅